### ISSUE TITLE
Poll: Put what service(s) you would like to see implemented here

### ISSUE THREAD

**Issue by studioph** [paulhutchings/bearboto3#6]
There are over 200 services in AWS and the boto3 library. Some are obviously more niche than others. If there is a comment with the service you're interested in, leave a thumbs up. If not, go ahead and comment with the service name.

It may take awhile for me to implement all the services people are looking for if I'm doing it by myself. Fortunately, this kind of work is easily parallelized! Consider making a PR to add support for the service you're interested in.

Also see the [list of services](https://github.com/paulhutchings/bearboto3/#supported-aws-services) to see what is currently supported.

**Comment by ivanistheone** [comment]
`ec2` seems like a logical next step

**Comment by studioph** [comment]
Indeed, that is in fact the one I am currently working on. It does have an awful lot of types to go through though, probably the most in the whole SDK. I did S3 by hand but EC2 is forcing me to figure out a way to automate the class annotations and unit test creation. The benefit is that adding future services should be much easier once I have a system.
